build-support/bin/generate_github_workflows.py

```diff
@@ -198,6 +198,38 @@ def checkout(
     return steps
 
 
+def launch_bazel_remote() -> Sequence[Step]:
+    return [
+        {
+            "name": "Launch bazel-remote",
+            "if": "github.repository_owner == 'pantsbuild'",
```

Comment:
Is this intending to be the "not from a fork" filter?

The GitHub docs about this field (https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) are ambiguous to me. https://github.com/orgs/community/discussions/26829#discussioncomment-3253575 suggests it might be "... the repository where the PR is in" which doesn't sound quite right, with the rather longer `github.event.pull_request.head.repo.full_name` being mentioned. Have you confirmed this one behaves as desired?

An alternative might be just checking whether the secret can be accessed, directly, e.g.

```suggestion
            "if": gha_expr('secrets.AWS_ACCESS_KEY_ID'),
```

Reply:
Good point, I tested it out and it is not correct!

But I will just go ahead and make the change to support read-only access when the secret isn't available, since that will touch this anyway.

FYI the expression syntax is not necessary in an `if` - GHA always interprets the clause as an expression.